main.go

```diff
@@ -50,25 +59,31 @@ func main() {
 
 	in := bufio.NewReader(os.Stdin)
 
-	for {
-		b, err := in.ReadByte()
-		if err != nil {
-			log.Printf("%v", err)
+	go func() {
+		for {
+			b, err := in.ReadByte()
+			if err != nil {
+				log.Printf("%v", err)
 
-			break
-		}
-		m.GetInputChan() <- b
+				break
+			}
+			m.GetInputChan() <- b
 
-		if len(m.GetInputChan()) > 0 {
-			if err := m.InjectSerialIRQ(); err != nil {
-				log.Printf("InjectSerialIRQ: %v", err)
+			if len(m.GetInputChan()) > 0 {
+				if err := m.InjectSerialIRQ(); err != nil {
+					log.Printf("InjectSerialIRQ: %v", err)
+				}
+			}
+
+			if before == 0x1 && b == 'x' {
+				break
```

Comment:
If the user inputs ctrl-a, x, then gokvm should be exited correctly.
So it would be preferable to revert the terminal settings and exit the process instead of break.

```suggestion
				restoreMode()
				os.Exit(0)
```